src/common/modals/AddInstance/Import.js

```diff
@@ -121,17 +121,17 @@ const Import = ({
       return;
     }
 
-    const modloader = [];
-    if (isForge) modloader.push(FORGE);
-    else if (isFabric) modloader.push(FABRIC);
-    else if (isVanilla) modloader.push(VANILLA);
+    const laoder = {};
```

Comment:
Typo.

Reply:
Actually, there's a lot of "laoder" in this PR.

Reply:
thanks for telling me that, I will fix that